lock-threads/lock-threads.yaml

```diff
@@ -0,0 +1,38 @@
+name: 'Lock Threads'
+
+on:
+  schedule:
+    - cron: '42 5 * * *' # 5:42 AM UTC every day
+  workflow_dispatch:
+
+permissions:
+  issues: write
+  # Enable the line below if you turn on pull request locking
+  # pull-requests: write
+
+concurrency:
+  group: lock-threads
+
+jobs:
+  action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/lock-threads@v4
+        with:
+          # Remove the line below to turn on pull request locking
+          process-only: 'issues'
+
+          # Issue-locking options
+          issue-inactive-days: 60
+          issue-comment: >
+            This issue has been automatically locked. If you believe you have
+            found a related problem, please open a new issue (with
+            [a reprex](https://reprex.tidyverse.org/)) and link to this issue.
+
+          # Pull request-locking options
+          pr-inactive-days: 60
+          pull-comment: >
+            This pull request has been automatically locked. If you believe
+            you have found a related problem, please open a new issue
+            (with [a reprex](https://reprex.tidyverse.org/)) and link to this
+            pull request.
```

Comment:
I wonder if this wiki article would fit this context better? https://github.com/rstudio/shiny/wiki/Writing-Good-Feature-Requests

Reply:
Maybe we could say _with a reprex or a feature request_ and link to those? I don't think commenting on old pull requests is as much of a problem (which is why I didn't enable locking PRs by default) but in most cases where people are commenting on these issues/PRs it's because they're running into a bug and the issue or PR is turning up in search. In other words, not usually for feature requests.

Reply:
Yea, and now that I think of it, I don't feel great about adopting this as a general policy (locking PR after 60 days or ever longer). I can maybe see adoption of this (with a longer inactive period) in newer repos that are under heavy active development, but this would lock many PRs that make useful contributions across many repos. I understand the desire to declare bankruptcy, but I also believe no clear response is better than a thought-less negative one.

Reply:
Just to clarify:

1. This action only locks __closed__ issues or PRs.
2. It only locks closed __issues__ as currently written and if installed as-is.

The settings are present to enable locking closed PRs after a period of inactivity, but that'd be something we turn on on a case-by-case basis or we'd decide to enable before deploying this in many places.

Reply:
Ahh, ok, I'm good with that

Reply:
And, to answer your question, yes, I think that makes sense to callout and provide different links for both situations (bug fix or feature request)